External/TestDataSet/analysis/DataParameters_LowQ.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
We can still add this as dataPar_LowQ.json - it is not forbidden to have alternative dataPars.

Reply:
Henk asked the same thing. If you want then sure. I thought we already have enough JSONs and a user could lose the overview.

Reply:
Added it here 7e0ddaa. I sticked to camel case though ;)